provisioning/container_images.go

```diff
@@ -30,6 +30,7 @@ type Images struct {
 	StaticIpManager              string `json:"baremetalStaticIpManager"`
 	IronicAgent                  string `json:"baremetalIronicAgent"`
 	ImageCustomizationController string `json:"imageCustomizationController"`
+	MachineOSImages              string `json:"machineOSImages"`
```

Comment:
This patch is missing the generated change to manifests/0000_31_cluster-baremetal-operator_01_images.configmap.yaml

Reply:
Done.